### DISCUSSION TITLE
Error 400: redirect_uri_mismatch

### DISCUSSION THREAD

**Discussion by KierenB** [YukiGasai/obsidian-google-calendar#127] [Q&A] [answered]
Hi there,

I followed the installation instructions and created the Google auth client as detailed in the directions. I set the origins and redirect URIs as shown (https://127.0.0.1:42813, https://127.0.0.1:42813/callback).

When I try to login I get the `Access blocked: This app’s request is invalid` error.

Any ideas where I went wrong, or how I can troubleshoot it?

I tested the auth client in a browser by typing `https://accounts.google.com/o/oauth2/auth?client_id={client_id}&response_type=token&redirect_uri=https://127.0.0.1:42813/callback&scope=https://www.googleapis.com/auth/calendar` and was able to authenticate so I know my app is working.

**Comment by YukiGasai** [comment, marked as answer]
Hi,
make sure the URL starts with `http` not with `https`.

**Reply by KierenB** [reply]
Thanks!
